lighthouse-core/test/report/html/renderer/util-test.js

```diff
@@ -152,7 +152,25 @@ describe('util helpers', () => {
 
       // Original audit results should be restored.
       const preparedResult = Util.prepareReportResult(clonedSampleResult);
+
+      // Delete stack packs from preparedResult; we test it separately below.
+      Object.values(preparedResult.audits).forEach(audit => {
+        if (audit.stackPacks) {
+          delete audit.stackPacks;
+        }
+      });
```

Comment:
I kind of hate we have to do this. See my proposal in #8633 for avoiding it altogether

Reply:
Removed now that that landed!